app/views/users/two_factor_authentication_setup/index.html.erb

```diff
@@ -79,4 +81,6 @@
 
 <%= render 'shared/cancel', link: destroy_user_session_path %>
 
+<%= link_to t('account_reset.cancel_request.cancel'), root_url %>
```

Comment:
Are we intending to add a second link to the live MFA setup screen?

![image](https://user-images.githubusercontent.com/1779930/161263513-aa105896-d899-4037-b0b1-3ae6a7db02dd.png)